library/Witch/src/Juvix/Witch/CPSTranslation/Transform.hs

```diff
@@ -0,0 +1,410 @@
+{-# LANGUAGE RecordWildCards, DeriveFunctor, DeriveAnyClass #-}
```

Comment:
I'd like to review this later when I can get this branch to build, as it'll help be investigate the types more.

Reply:
I'm fixing it, sorry for the mess.